src/app/space-settings/areas/create-area-dialog/create-area-dialog.component.ts

```diff
@@ -4,6 +4,10 @@ import { Modal } from 'ngx-modal';
 import { Context, AreaService, Area, AreaAttributes } from 'ngx-fabric8-wit';
 import { ContextService } from '../../../shared/context.service';
 
+interface Errors {
+  unique: boolean
```

Comment:
Nevermind my comment, I was confusing errors. I agree with @dgutride that the interface should be moved to the models folder.